src/cookies.js

```diff
@@ -247,19 +247,23 @@ function checkOriginForSettingCookie(win, options, name) {
     );
     return;
   }
+  const current = parseUrlDeprecated(win.location.href).hostname.toLowerCase();
+  const proxy = parseUrlDeprecated(urls.cdn).hostname.toLowerCase();
```

Comment:
Let's move these back to their original positions. They don't need to be evaluated if the first assert fails.